dcrwallet.go

```diff
@@ -230,6 +230,7 @@ func run(ctx context.Context) error {
 		loader.RunAfterLoad(func(w *wallet.Wallet) {
 			rpcserver.StartWalletService(gRPCServer, w)
 			rpcserver.StartVotingService(gRPCServer, w)
+			rpcserver.StartDecodeMessageService(gRPCServer, w)
```

Comment:
this service can be started as soon as the server starts, instead of only starting it after the wallet is loaded, since it does not require the wallet